src/megatron/bridge/models/ernie_vl/ernie45_vl_bridge.py

```
@@ -0,0 +1,835 @@
+    tensor of shape ``[2, num_experts]`` where row 0 is the text pool bias and
+    row 1 is the vision pool bias.  This mapping extracts the appropriate row
+    based on ``slice_name``.
+
+    For export (megatron_to_hf), the text mapping buffers its bias in a
+    class-level dict keyed by resolved HF param name.  The vision mapping
+    retrieves the buffered text bias, stacks them into ``[2, N]``, and
+    returns the merged tensor.  This ensures only one entry per HF key.
+    """
+
+    # Class-level buffer: {resolved_hf_param: text_bias_tensor}
+    _export_buffer: Dict[str, torch.Tensor] = {}
```

Ruff (0.15.9)

[warning] Mutable default value for class attribute

(RUF012)